packages/manager/apps/pci-rancher/src/pages/index.tsx

```diff
@@ -16,7 +16,7 @@ export default function Home() {
   const { data, isError, error, isLoading } = useRanchers();
 
   if (isError) {
-    return <ErrorBanner error={error} />;
+    return <ErrorBanner error={error.response} />;
```

Comment:
Here since you're not testing that error is truthy, I would use the optional chaining
```suggestion
    return <ErrorBanner error={error?.response} />;
```
But it may be unnecessary if error is necessarily defined when isError is true